README.md

```diff
@@ -2654,13 +2654,18 @@ Conditional Types are a difficult topic to get around so here are some extra res
 
 ### Troubleshooting Handbook: Utilities
 
-These are all built in, [see source in es5.d.ts](https://github.com/microsoft/TypeScript/blob/2c458c0d1ccb96442bca9ce43aa987fb0becf8a9/src/lib/es5.d.ts#L1401-L1474):
+These are all built in, [see source in es5.d.ts](https://github.com/microsoft/TypeScript/blob/main/src/lib/es5.d.ts#L1523-L1637):
```

Comment:
Sounds good to "bump" this reference, but let's use a commit instead of `main`, since the main branch might change and invalidate the line range. Sounds good?

Reply:
@filiptammergard Oh my bad, you're right. Updated, please check again